src/main/java/Events/RandomEvent.java

```diff
@@ -0,0 +1,87 @@
+package Events;
+
+import Bot.DiscordBot;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+import javax.annotation.Nonnull;
+import java.awt.*;
+import java.util.Random;
+
+public class RandomEvent extends ListenerAdapter {
+    private EmbedBuilder card = new EmbedBuilder();
+
+    @Override
+    public void onGuildMessageReceived(@Nonnull GuildMessageReceivedEvent event) {
+        card.setColor(Color.BLUE);
+        String[] args = event.getMessage().getContentRaw().split("//s+");
+
+        if(!args[0].startsWith(DiscordBot.PREFIX)){ //If the prefix is not present, end method and do nothing
```

Comment:
For the next update we'll do we need to add static on-demand import on the prefix, but ill approve